utils/coord2area_def.py

```diff
@@ -17,21 +17,46 @@
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
 """Convert human coordinates (lon and lat) to an area definition.
 
-Here is a usage example:
+Here is a usage example.
+
 python coord2area_def.py france stere 42.0 51.5 -5.5 8.0 1.5
-(the arguments are "name proj min_lat max_lat min_lon max_lon resolution(km)")
+The arguments are "name proj min_lat max_lat min_lon max_lon resolution(km)".
+The command above yelds the following result.
+
+### +proj=stere +lat_0=46.75 +lon_0=1.25 +ellps=WGS84
+
+france:
+  description: france
+  projection:
+    proj: stere
+    ellps: WGS84
+    lat_0: 46.75
+    lon_0: 1.25
+  shape:
+    height: 703
+    width: 746
+  area_extent:
+    lower_left_xy: [-559750.381098, -505020.675776]
+    upper_right_xy: [559750.381098, 549517.351948]
+
+
+The first commented line is just a sum-up. The value of "description" can be changed to any descriptive text.
+
+Such a custom yaml configuration can be profitably saved in a local areas.yaml configuration file that won't be
+overridden by future updates of SatPy package. For that purpose the local processing script may have suitable
+lines as reported below.
+
+# set PPP_CONFIG_DIR for custom composites
+import os
+os.environ['PPP_CONFIG_DIR'] = '/my_local_path/for_satpy_configuration'
+
+As a further functionality this script may give a quick display of the defined area,
+provided the path for the GSHHG library is supplied via the "-s" option.
```

Comment:
It should maybe be mentioned that pycoast, pillow and aggdraw need to be installed also for this functionality to work.

Reply:
Just done. Thanks for the suggestion!